src/containers/FooterContainer.vue

```diff
@@ -1,12 +1,62 @@
 <template>
-    <footer>
-    </footer>
+  <footer>
+    <Dialog
+      header="Header"
+      footer="Footer"
+      v-model:visible="display"
+      :style="{ width: '50vw', height: '65vh', justifyContent: 'space-between' }"
```

Comment:
It's better to remove this height, since you can see the background through the 'slots' when the dialog is too high